src/colors.js

```diff
@@ -166,6 +166,35 @@ const primary = {
     800: "#225923",
     900: "#1d4a1f",
   },
+  grayBlue: {
+    25: "#fcfcfd",
+    50: "#f8f9fc",
+    100: "#eaecf5",
+    200: "#d5d9eb",
+    300: "#b3b8db",
+    400: "#717bbc",
+    500: "#4e5ba6",
+    600: "#3e4784",
+    700: "#363f72",
+    800: "#293056",
+    850: "#1B243F",
```

Comment:
These are the latest color values from 850 onwards:

```suggestion
    850: "#202946",
    875: "#10172C",
    900: "#0f0d2c",
    950: "#080f25",
    975: "#03091B",
```